src/CLI/ErrorCollector.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace AxeWindowsScanner
+{
+    public class ErrorCollector : IErrorCollector
+    {
+        public IReadOnlyList<string> ParameterErrors => _parameterErrors;
+
+        public IReadOnlyList<string> ScanErrors => _scanErrors;
```

Comment:
I notice this property is not defined in the IErrorCollector interface. I'm not sure if that's intentional.

Reply:
Nice catch--this was an early design thought that I later abandoned (but failed to clean up completely)